travis/git-modif

```diff
@@ -41,13 +41,16 @@ def main():
     subprocess.check_output(["git", "fetch", "origin", "{0}:{0}".foramt(os.environ["MAIN_BRANCH"])])
```

Comment:
What does this script do ? no comment. Seems to be be never called.
Note the ```foramt``` on this line.
Should it be removed ?

Reply:
It return a "bash boolean" it the given folder has some changes from the reference branch.
effectively it's no more used because it don't works well, then I will remove it :-)